.github/workflows/create-charts.sh

```diff
@@ -32,6 +32,12 @@ update_data_services_charts() {
         echo '  - name: mongodb'
         echo '    repository:' $HELM_GCS_REPO
         echo '    version:' $mongodb_version
+        echo '    condition: mongodb.enabled'
+        echo '  - name: postgresql'
+        echo '    repository:' '"https://charts.bitnami.com/bitnami"'
+        echo '    version:' 10.2.6
+        echo '    condition: postgresql.enabled'
```

Comment:
```suggestion
        echo '    condition: postgres.enabled'
```

Reply:
it should stay `postgresql` here as explained in above comment.